doc/cascadia/SettingsSchema.md

```diff
@@ -33,6 +33,7 @@ Properties listed below are specific to each unique profile.
 | `padding` | _Required_ | String | `0, 0, 0, 0` | Sets the padding around the text within the window. Can have three different formats: `"#"` sets the same padding for all sides, `"#, #"` sets the same padding for left-right and top-bottom, and `"#, #, #, #"` sets the padding individually for left, top, right, and bottom. |
 | `snapOnInput` | _Required_ | Boolean | `true` | When set to `true`, the window will scroll to the command input line when typing. When set to `false`, the window will not scroll when you start typing. |
 | `startingDirectory` | _Required_ | String | `%USERPROFILE%` | The directory the shell starts in when it is loaded. |
+| `startingTitle` | Optional | String | | The title to pass to the shell on startup. Some shells (like `bash`) may choose to ignore this initial value, while others (`cmd`, `powershell`) may use this value over the lifetime of the application. |

```

Comment:
default should be empty string

Should we add a note at the bottom of the doc saying what the defaults are for the generated profiles? Or maybe that's something for a completely different user doc?

Reply:
Also maybe specifying the specific difference between `startingTitle` and `tabTitle`?

Reply:
Default is _unset_, not _empty string_. Distinction!